MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,569 @@
+---
+layout: post
+title: Toolbar Customization in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here all about toolbar customization support in the Syncfusion .NET MAUI ImageEditor (SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Toolbar Customization in .NET MAUI Image Editor (SfImageEditor)
+
+You can customize the existing toolbar appearance, visibility, position, and orientation of each toolbar and can include new toolbars.
+
+## Toolbar style customization
+
+To customize the appearance of the toolbar by using the `ToolbarSettings` property. Below are some of the properties you can customize:
+
+* `Background`: Specifies background color of the toolbar.
+* `IconColor`: Specifies the color of the icons in the toolbar.
+* `IconSize`: Specifies the size of the icons in the toolbar. The default value is 44.
+* `DisabledColor`: Specifies the color of the disabled icons in the toolbar.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="table.png">
```

Comment:
IconSize="50" => is this working

IconSize="50, 50"

Reply:
IconSize = 50 => its working.
I added IconSize="50, 50"